i18n/l10n/sv.json

```diff
@@ -248,73 +248,65 @@
   "pwfield.good": "Ganska starkt lösenord",
   "pwfield.repeat_different": "Lösenorden är olika",
-  "pwfield.repeat_password": "Repetera lösenordet",
   "pwfield.strong": "Starkt lösenord",
   "pwfield.terrible": "Extremt svagt lösenord",
   "pwfield.weak": "Svagt lösenord",
+  "register.create-account": "Skapa ett eduID med din e-postadress för att börja.",
   "required": "Obligatorisk",
   "resend.button": "Skicka ett nytt verifieringsmeddelande",
-  "resend.subtitle": "Skicka ett verifieringsmeddelande till den här adressen",
-  "resend.title": "E-postadressen används redan",
-  "security.account_description": "Använd knappen nedan för att permanent ta bort ditt eduID-konto.",
-  "security.account_title": "Kontoborttagning",
-  "security.add_u2f_token": "Lägg till Säkerhetsnyckel",
+  "resend.email-label": "Slutför skapandet av eduID genom att klicka länken skickad till:",
+  "resend.link-sent": "En länk har skickats till din e-postadress.",
+  "security.add_u2f_token": "Lägg till säkerhetsnyckel",
   "security.add_webauthn_token_device": "Registrera den här enheten som en Säkerhetsnyckel",
-  "security.add_webauthn_token_key": "Registrera en Säkerhetsnyckel",
-  "security.change_info": "Du måste logga in igen för att byta lösenord.",
-  "security.change_password": "Byt lösenord",
-  "security.confirm_button": "eduID-kontot kommer att tas bort",
-  "security.confirm_title": "Ta bort konto",
-  "security.confirm_title_chpass": "Byt lösenord",
-  "security.confirm_title_deletion": "Kontoborttagning",
+  "security.add_webauthn_token_key": "Lägg till en säkerhetsnyckel",
+  "security.button_delete_account": "Ta bort mitt eduID",
   "security.creation_date": "Skapat den",
   "security.credential": "Lösenord",
-  "security.delete_account": "Ta bort eduID-kontot",
-  "security.description": "Beskrivning",
-  "security.for-second-factor": "Registrera Säkerhetsnycklar för autentisering hos eduID",
-  "security.last_used": "Användes senast",
-  "security.long_description": "Här kan du byta lösenord för ditt eduID-konto.",
+  "security.description": "Namn",
+  "security.last-used.date": "Aldrig använt",
+  "security.last_used": "Senast använt",
   "security.main_title": "Säkerhet",
-  "security.modal_info": "Är du säker på att du vill ta bort ditt eduID-konto? All information knuten till dig och ditt kontot kommer att tas bort.",
-  "security.modal_notes": "Av säkerhetsskäl så måste du logga in igen för att ta bort kontot.",
   "security.password_credential_type": "Lösenord",
   "security.remove": "Ta bort",
-  "security.u2f-describe-title": "Ange en beskrivning av Säkerhetsnyckeln du vill registrera.",
+  "security.second-factor": "Lägg till ett extra sätt, utöver email och lösenord, för att kunna identifiera dig så du är säker på att bara du har tillgång till ditt eduID.",
+  "security.security-key_title": "Gör ditt eduID säkrare",
+  "security.u2f-describe-title": "Ge säkerhetsnyckeln ett namn så du kan skilja dem åt om du lägger till fler.",
```

Comment:
We should choose Säkerhetsnyckel or säkerhetsnyckel. I think we made the choice for capital S as that was the way Google does it. That doesn't mean we have to do the same choice as long as we are consistent :)

Reply:
I did a quick google [here](https://support.google.com/accounts/answer/6103523?co=GENIE.Platform%3DAndroid&hl=en) (Eng) and [here](https://support.google.com/accounts/answer/6103523?co=GENIE.Platform%3DAndroid&hl=sv) (Swe) and could not find the capitalised use, so I will move forward with lowercase...of course do let me know if these are not the right sources.